main.go

```diff
@@ -358,6 +380,22 @@ func main() {
 		}
 	}()
 
+	// tracer sink
+	tracer, err := tracer.GetSink(cfg.Controller.Tracer)
```

Comment:
It feels weird to me we have the enable flags and do not use them to control the initialization in the main 🤔 

Why aren't we doing this?